### PR TITLE
[KARAF-4236] Cellar DNS nodes discovery

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @jbonofre

I gonna add the corresponding `cellar-dns` feature. Even that, LGTM.

File: pom.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
         <felix.webconsole.version>4.2.14</felix.webconsole.version>
         <hazelcast.version>3.5.4</hazelcast.version>
         <jclouds.version>1.9.1</jclouds.version>
+        <spotify-dns.version>3.0.3</spotify-dns.version>
```
Comment (lburgazzoli) [inline pom.xml]: I'm unsure this is needed, I'm going to clean it up

**Comment by lburgazzoli** [comment]
@jbonofre would you like me to do so ?

**Comment by jbonofre** [comment]
@lburgazzoli up to you: I planned to checkout, fix and squash locally. So let me know.

**Comment by lburgazzoli** [comment]
It should be ready to merge now, check to see if I made any mistake ;)

**Comment by jbonofre** [comment]
Reviewing. Thanks !

**Pull request merged**